packages/row/src/css/index.js

```diff
@@ -101,13 +104,18 @@ export default {
     lineHeight: 0,
     cursor: 'pointer',
     border: 'none',
-    color: core.colors.gray02,
     background: 'none',
-    transition: `all ${core.motion.speedNormal}`,
+    transition: `all ${core.motion.speedNormal}`
+  },
+  [`.psds-row__action-bar__button--theme-${themeNames.dark}`]: {
```

Comment:
We have traditionally tried to refer to `defaultTheme` from the `theme` package instead of `themeNames.dark`.  I frankly can't think of the reasoning that went into this decision in the first place -- it's just something I've tried to be consistent on since.  

Are there any pros/cons that would push us one way or the other besides internal consistency that occur to you?

Reply:
well....i've probably broken that convention a couple times :/

I don't have a good sense either way. i'll update to use `defaultTheme` and match convention